app/javascript/utils/form/final_form/MultiDateField.js

```diff
@@ -40,53 +40,65 @@ export const MultiDateField = ({
   })
   let errorField
   if (formName) {
-    errorField = form.getState().errors[formName] ? form.getState().errors[formName][index] : null
+    if (form.getState().errors[formName]) {
```

Comment:
This error stuff is super complicated, I wonder if there are react final form error handling best practices we can start to apply across the app (not saying anything needs to change now)

Reply:
agreed!